share/picongpu/tests/CurrentDeposition/lib/python/test/CurrentDeposition/AfuncW.py

```diff
@@ -0,0 +1,107 @@
+import numpy as np

```

Comment:
This file misses a general deocumentation at the beginning explaining its purpose. (just few lines)

Reply:
done